src/sql/src/statements/insert.rs

```diff
@@ -87,8 +110,14 @@ fn sql_exprs_to_values(exprs: &Vec<Vec<Expr>>) -> Result<Vec<Vec<Value>>> {
                 Expr::Value(v) => v.clone(),
                 Expr::Identifier(ident) => {
                     if ident.quote_style.is_none() {
-                        Value::Placeholder(ident.value.clone())
+                        // Special processing for `default` value
+                        if ident.value.to_lowercase() == "default" {
+                            Value::Placeholder(ident.value.clone())
+                        } else {
+                            return None;
+                        }
                     } else {
+                        // Identifier with double quotes, we treat it as string.
```

Comment:
nit: 
```suggestion
                        // Identifier with single quotes, we treat it as string.
```

Reply:
It's processing `identifier` with double quotes here, so the comment is correct. We treat the double quoted identifiers in `values` as strings.